integration-tests/testModifyQuestionAndAnswers.py

```diff
@@ -1,7 +1,7 @@
 import api
 import sys
 
-reg = api.register("aloy", "chieftain")
+reg = api.auth("aloy", "chieftain")
```

Comment:
I believe the username does not exist when the integration tests are run. Using `api.register` should do it 🙂

Reply:
it's fixed